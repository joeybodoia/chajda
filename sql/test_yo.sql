CREATE EXTENSION IF NOT EXISTS pspacy;
SELECT spacy_lemmatize('yo',' "Eyi ni idajọ idanwo mi ti Mo n fi sinu Tumọ-ọrọ Google lati ṣe agbekalẹ awọn ọran idanwo."' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>True , add_positions=>True);
SELECT spacy_lemmatize('yo',' "Eyi ni idajọ idanwo mi ti Mo n fi sinu Tumọ-ọrọ Google lati ṣe agbekalẹ awọn ọran idanwo."' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>True , add_positions=>False);
SELECT spacy_lemmatize('yo',' "Eyi ni idajọ idanwo mi ti Mo n fi sinu Tumọ-ọrọ Google lati ṣe agbekalẹ awọn ọran idanwo."' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>False , add_positions=>True);
SELECT spacy_lemmatize('yo',' "Eyi ni idajọ idanwo mi ti Mo n fi sinu Tumọ-ọrọ Google lati ṣe agbekalẹ awọn ọran idanwo."' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>False , add_positions=>False);
SELECT spacy_lemmatize('yo',' "Eyi ni idajọ idanwo mi ti Mo n fi sinu Tumọ-ọrọ Google lati ṣe agbekalẹ awọn ọran idanwo."' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>True , add_positions=>True);
SELECT spacy_lemmatize('yo',' "Eyi ni idajọ idanwo mi ti Mo n fi sinu Tumọ-ọrọ Google lati ṣe agbekalẹ awọn ọran idanwo."' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>True , add_positions=>False);
SELECT spacy_lemmatize('yo',' "Eyi ni idajọ idanwo mi ti Mo n fi sinu Tumọ-ọrọ Google lati ṣe agbekalẹ awọn ọran idanwo."' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>False , add_positions=>True);
SELECT spacy_lemmatize('yo',' "Eyi ni idajọ idanwo mi ti Mo n fi sinu Tumọ-ọrọ Google lati ṣe agbekalẹ awọn ọran idanwo."' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>False , add_positions=>False);
SELECT spacy_lemmatize('yo',' "Eyi ni idajọ idanwo mi ti Mo n fi sinu Tumọ-ọrọ Google lati ṣe agbekalẹ awọn ọran idanwo."' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>True , add_positions=>True);
SELECT spacy_lemmatize('yo',' "Eyi ni idajọ idanwo mi ti Mo n fi sinu Tumọ-ọrọ Google lati ṣe agbekalẹ awọn ọran idanwo."' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>True , add_positions=>False);
SELECT spacy_lemmatize('yo',' "Eyi ni idajọ idanwo mi ti Mo n fi sinu Tumọ-ọrọ Google lati ṣe agbekalẹ awọn ọran idanwo."' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>False , add_positions=>True);
SELECT spacy_lemmatize('yo',' "Eyi ni idajọ idanwo mi ti Mo n fi sinu Tumọ-ọrọ Google lati ṣe agbekalẹ awọn ọran idanwo."' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>False , add_positions=>False);
SELECT spacy_lemmatize('yo',' "Eyi ni idajọ idanwo mi ti Mo n fi sinu Tumọ-ọrọ Google lati ṣe agbekalẹ awọn ọran idanwo."' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>True , add_positions=>True);
SELECT spacy_lemmatize('yo',' "Eyi ni idajọ idanwo mi ti Mo n fi sinu Tumọ-ọrọ Google lati ṣe agbekalẹ awọn ọran idanwo."' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>True , add_positions=>False);
SELECT spacy_lemmatize('yo',' "Eyi ni idajọ idanwo mi ti Mo n fi sinu Tumọ-ọrọ Google lati ṣe agbekalẹ awọn ọran idanwo."' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>False , add_positions=>True);
SELECT spacy_lemmatize('yo',' "Eyi ni idajọ idanwo mi ti Mo n fi sinu Tumọ-ọrọ Google lati ṣe agbekalẹ awọn ọran idanwo."' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>False , add_positions=>False);
SELECT spacy_tsvector('yo',' "Eyi ni idajọ idanwo mi ti Mo n fi sinu Tumọ-ọrọ Google lati ṣe agbekalẹ awọn ọran idanwo."');
