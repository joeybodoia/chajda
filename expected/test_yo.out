CREATE EXTENSION IF NOT EXISTS pspacy;
NOTICE:  extension "pspacy" already exists, skipping
SELECT spacy_lemmatize('yo',' "Eyi ni idajọ idanwo mi ti Mo n fi sinu Tumọ-ọrọ Google lati ṣe agbekalẹ awọn ọran idanwo."' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>True , add_positions=>True);
                                              spacy_lemmatize                                              
-----------------------------------------------------------------------------------------------------------
   eyi:2 idajọ:4 idanwo:5 fi:10 sinu:11 tumọọrọ:12 google:13 lati:14 agbekalẹ:16 awọn:17 ọran:18 idanwo:19
(1 row)

SELECT spacy_lemmatize('yo',' "Eyi ni idajọ idanwo mi ti Mo n fi sinu Tumọ-ọrọ Google lati ṣe agbekalẹ awọn ọran idanwo."' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>True , add_positions=>False);
                             spacy_lemmatize                              
--------------------------------------------------------------------------
   eyi idajọ idanwo fi sinu tumọọrọ google lati agbekalẹ awọn ọran idanwo
(1 row)

SELECT spacy_lemmatize('yo',' "Eyi ni idajọ idanwo mi ti Mo n fi sinu Tumọ-ọrọ Google lati ṣe agbekalẹ awọn ọran idanwo."' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>False , add_positions=>True);
                                                             spacy_lemmatize                                                             
-----------------------------------------------------------------------------------------------------------------------------------------
   eyi:2 ni:3 idajọ:4 idanwo:5 mi:6 ti:7 mo:8 n:9 fi:10 sinu:11 tumọọrọ:12 google:13 lati:14 ṣe:15 agbekalẹ:16 awọn:17 ọran:18 idanwo:19
(1 row)

SELECT spacy_lemmatize('yo',' "Eyi ni idajọ idanwo mi ti Mo n fi sinu Tumọ-ọrọ Google lati ṣe agbekalẹ awọn ọran idanwo."' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>False , add_positions=>False);
                                      spacy_lemmatize                                      
-------------------------------------------------------------------------------------------
   eyi ni idajọ idanwo mi ti mo n fi sinu tumọọrọ google lati ṣe agbekalẹ awọn ọran idanwo
(1 row)

SELECT spacy_lemmatize('yo',' "Eyi ni idajọ idanwo mi ti Mo n fi sinu Tumọ-ọrọ Google lati ṣe agbekalẹ awọn ọran idanwo."' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>True , add_positions=>True);
                                                         spacy_lemmatize                                                          
----------------------------------------------------------------------------------------------------------------------------------
   ":2 eyi:3 idajọ:5 idanwo:6 fi:11 sinu:12 tumọ:13 -:14 ọrọ:15 google:16 lati:17 agbekalẹ:19 awọn:20 ọran:21 idanwo:22 .:23 ":24
(1 row)

SELECT spacy_lemmatize('yo',' "Eyi ni idajọ idanwo mi ti Mo n fi sinu Tumọ-ọrọ Google lati ṣe agbekalẹ awọn ọran idanwo."' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>True , add_positions=>False);
                                  spacy_lemmatize                                  
-----------------------------------------------------------------------------------
   " eyi idajọ idanwo fi sinu tumọ - ọrọ google lati agbekalẹ awọn ọran idanwo . "
(1 row)

SELECT spacy_lemmatize('yo',' "Eyi ni idajọ idanwo mi ti Mo n fi sinu Tumọ-ọrọ Google lati ṣe agbekalẹ awọn ọran idanwo."' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>False , add_positions=>True);
                                                                         spacy_lemmatize                                                                         
-----------------------------------------------------------------------------------------------------------------------------------------------------------------
   ":2 eyi:3 ni:4 idajọ:5 idanwo:6 mi:7 ti:8 mo:9 n:10 fi:11 sinu:12 tumọ:13 -:14 ọrọ:15 google:16 lati:17 ṣe:18 agbekalẹ:19 awọn:20 ọran:21 idanwo:22 .:23 ":24
(1 row)

SELECT spacy_lemmatize('yo',' "Eyi ni idajọ idanwo mi ti Mo n fi sinu Tumọ-ọrọ Google lati ṣe agbekalẹ awọn ọran idanwo."' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>False , add_positions=>False);
                                          spacy_lemmatize                                           
----------------------------------------------------------------------------------------------------
   " eyi ni idajọ idanwo mi ti mo n fi sinu tumọ - ọrọ google lati ṣe agbekalẹ awọn ọran idanwo . "
(1 row)

SELECT spacy_lemmatize('yo',' "Eyi ni idajọ idanwo mi ti Mo n fi sinu Tumọ-ọrọ Google lati ṣe agbekalẹ awọn ọran idanwo."' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>True , add_positions=>True);
                                              spacy_lemmatize                                              
-----------------------------------------------------------------------------------------------------------
   Eyi:2 idajọ:4 idanwo:5 fi:10 sinu:11 Tumọọrọ:12 Google:13 lati:14 agbekalẹ:16 awọn:17 ọran:18 idanwo:19
(1 row)

SELECT spacy_lemmatize('yo',' "Eyi ni idajọ idanwo mi ti Mo n fi sinu Tumọ-ọrọ Google lati ṣe agbekalẹ awọn ọran idanwo."' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>True , add_positions=>False);
                             spacy_lemmatize                              
--------------------------------------------------------------------------
   Eyi idajọ idanwo fi sinu Tumọọrọ Google lati agbekalẹ awọn ọran idanwo
(1 row)

SELECT spacy_lemmatize('yo',' "Eyi ni idajọ idanwo mi ti Mo n fi sinu Tumọ-ọrọ Google lati ṣe agbekalẹ awọn ọran idanwo."' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>False , add_positions=>True);
                                                             spacy_lemmatize                                                             
-----------------------------------------------------------------------------------------------------------------------------------------
   Eyi:2 ni:3 idajọ:4 idanwo:5 mi:6 ti:7 Mo:8 n:9 fi:10 sinu:11 Tumọọrọ:12 Google:13 lati:14 ṣe:15 agbekalẹ:16 awọn:17 ọran:18 idanwo:19
(1 row)

SELECT spacy_lemmatize('yo',' "Eyi ni idajọ idanwo mi ti Mo n fi sinu Tumọ-ọrọ Google lati ṣe agbekalẹ awọn ọran idanwo."' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>False , add_positions=>False);
                                      spacy_lemmatize                                      
-------------------------------------------------------------------------------------------
   Eyi ni idajọ idanwo mi ti Mo n fi sinu Tumọọrọ Google lati ṣe agbekalẹ awọn ọran idanwo
(1 row)

SELECT spacy_lemmatize('yo',' "Eyi ni idajọ idanwo mi ti Mo n fi sinu Tumọ-ọrọ Google lati ṣe agbekalẹ awọn ọran idanwo."' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>True , add_positions=>True);
                                                         spacy_lemmatize                                                          
----------------------------------------------------------------------------------------------------------------------------------
   ":2 Eyi:3 idajọ:5 idanwo:6 fi:11 sinu:12 Tumọ:13 -:14 ọrọ:15 Google:16 lati:17 agbekalẹ:19 awọn:20 ọran:21 idanwo:22 .:23 ":24
(1 row)

SELECT spacy_lemmatize('yo',' "Eyi ni idajọ idanwo mi ti Mo n fi sinu Tumọ-ọrọ Google lati ṣe agbekalẹ awọn ọran idanwo."' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>True , add_positions=>False);
                                  spacy_lemmatize                                  
-----------------------------------------------------------------------------------
   " Eyi idajọ idanwo fi sinu Tumọ - ọrọ Google lati agbekalẹ awọn ọran idanwo . "
(1 row)

SELECT spacy_lemmatize('yo',' "Eyi ni idajọ idanwo mi ti Mo n fi sinu Tumọ-ọrọ Google lati ṣe agbekalẹ awọn ọran idanwo."' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>False , add_positions=>True);
                                                                         spacy_lemmatize                                                                         
-----------------------------------------------------------------------------------------------------------------------------------------------------------------
   ":2 Eyi:3 ni:4 idajọ:5 idanwo:6 mi:7 ti:8 Mo:9 n:10 fi:11 sinu:12 Tumọ:13 -:14 ọrọ:15 Google:16 lati:17 ṣe:18 agbekalẹ:19 awọn:20 ọran:21 idanwo:22 .:23 ":24
(1 row)

SELECT spacy_lemmatize('yo',' "Eyi ni idajọ idanwo mi ti Mo n fi sinu Tumọ-ọrọ Google lati ṣe agbekalẹ awọn ọran idanwo."' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>False , add_positions=>False);
                                          spacy_lemmatize                                           
----------------------------------------------------------------------------------------------------
   " Eyi ni idajọ idanwo mi ti Mo n fi sinu Tumọ - ọrọ Google lati ṣe agbekalẹ awọn ọran idanwo . "
(1 row)

SELECT spacy_tsvector('yo',' "Eyi ni idajọ idanwo mi ti Mo n fi sinu Tumọ-ọrọ Google lati ṣe agbekalẹ awọn ọran idanwo."');
                                                     spacy_tsvector                                                     
------------------------------------------------------------------------------------------------------------------------
 'agbekalẹ':16 'awọn':17 'eyi':2 'fi':10 'google':13 'idajọ':4 'idanwo':5,19 'lati':14 'sinu':11 'tumọọrọ':12 'ọran':18
(1 row)

